packages/styles/src/theming/common/variant-management/_sap_horizon.scss

```diff
@@ -0,0 +1,7 @@
+:root {
+  --fdVariantManagementHeadingColor: var(--sapLinkColor);
+  --fdVariantManagementHeadingHoverColor: var(--sapLink_Hover_Color);
+  --fdVariantManagementHeadingActiveColor: var(--sapLink_Active_Color);
+  --fdVariantManagementDirtyLabelColor: var(--sapGroup_TitleTextColor);
+  --fdVariantManagementCustomDirtyLabelSize: var(--sapFontSmallSize);
```

Comment:
`--fdVariantManagementCustomDirtyLabelSize: var(--sapFontSmallSize);` is the same in Quartz and Horizon. Why is it in the theming folder?